[ES, PT, JA, KO, RU, ZH] Translations - documentation

ilyaspiridonov opened the pull request:
Translated by Alconost

ilyaspiridonov commented:
@devnook Please take a look

devnook commented on site/es/docs/privacy-sandbox/overview/index.md:
```diff
@@ -0,0 +1,105 @@
+---
+layout: layouts/doc-post.njk
+title: What is the Privacy Sandbox?
+subhead: The Privacy Sandbox is a series of proposals to satisfy cross-site use cases without third-party cookies or other tracking mechanisms.
```
This text should be translated

devnook commented on site/es/docs/privacy-sandbox/overview/index.md:
```diff
@@ -0,0 +1,105 @@
+layout: layouts/doc-post.njk
+title: What is the Privacy Sandbox?
+subhead: The Privacy Sandbox is a series of proposals to satisfy cross-site use cases without third-party cookies or other tracking mechanisms.
+description: "What's in it, how to get involved, and what it's for."
```
This text should be translated

devnook commented on site/ja/docs/privacy-sandbox/attribution-reporting-migration/index.md:
```diff
@@ -0,0 +1,141 @@
+---
+layout: 'layouts/doc-post.njk'
+title: コンバージョン測定APIからアトリビューションレポートAPIへの移行ガイド（Chrome 92）
+subhead: |2-
```
Unnecessary line + 2-

devnook commented on site/ja/docs/privacy-sandbox/overview/index.md:
```diff
@@ -0,0 +1,106 @@
+---
+layout: layouts/doc-post.njk
+title: What is the Privacy Sandbox?
+subhead: The Privacy Sandbox is a series of proposals to satisfy cross-site use cases without third-party cookies or other tracking mechanisms.
```
Missing translation. Also on all other languages in this url.

devnook commented:
There are 2 translation strings missing from each language. LGTM to eveything else.

ilyaspiridonov commented:
> There are 2 translation strings missing from each language. LGTM to eveything else.

Thank you. It appears this was caused by a tab in frontmatter in [this English file]:(https://github.com/GoogleChrome/developer.chrome.com/blob/main/site/en/docs/privacy-sandbox/overview/index.md)
I've added a commit in this PR to fix that. 
The translations for these strings are also added now (had to request those separately, hence the delay).

devnook commented:
lgtm
